test/edu/csus/ecs/pc2/imports/ccs/ContestSnakeYAMLLoaderTest.java

```diff
@@ -3674,5 +3674,34 @@ private IInternalContest fullLoadSampleContest(String sampleName) throws Excepti
         return contest;
     }
     
+    
+    /**
+     * Test halt-contest-clock-at-set to true.
+     * 
+     * @throws Exception
+     */
+    public void testisHaltContestAtTimeZero() throws Exception {
+        String sampleContestDirName = "ccs1";
+        String dirname = getContestSampleCDPConfigDirname(sampleContestDirName);
+        
+        IInternalContest contest = snake.fromYaml(null, dirname, false);
+        assertNotNull("Expecting to load ccs1 contest",contest);
+        assertTrue("Expected halt at end of contest ", contest.getContestInformation().isAutoStartContest());
```

Comment:
This line is checking `isAutoStartContest()`.  I don't see how "auto START contest" is related to what the method claims to be testing -- which is that autoSTOP is correctly set.

Reply:
typo - fixing.